src/app/components/constitution-page/constitution.component.html

```diff
@@ -20,6 +20,7 @@ <h1> {{constitution.name}} </h1>
 			<td> {{numberOfCurrentUserFavorites()}} / {{numberOfMaxFavorites()}} </td>
 		</tr>
 	</table>
+	<span> {{getRemainingTimeMsg()}} </span>
```

Comment:
Je verrais bien une ligne en plus pour séparer un peu
```suggestion
    <br>
	<span> {{getRemainingTimeMsg()}} </span>
```

Reply:
Un <br> est peut-être trop, j'ai proposé un entre deux